### PR TITLE
SEO-175114 .NET-MAUI Maps External Linking

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Girija-Rajendran,

|Title|Description|
|--|--|
|Task Category| .NET MAUI Maps|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|External Linking PR link| External linking -https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-maps/pull/2|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175114/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Maps-Money-Keywords.aspx|

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui maps|0-10 |3FT|To improve our FT experience |added CSS classes alignment format, images aligned in zigzag form, added documentation, code example,renamed images & removed unwanted space |
|.net maui maps example|0-10 |3UG,4FT|To improve page experience | demo link added|
|.net maui maps control |0-10 |4FT|To improve our FT experience | improved on page changes|
|.net maui google maps |0-100|6FT |To improve FT ranking to 1st position| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|
|.net maui maps custom pin|0-10 |5UG| To improve our UG experience |added corresponding documentation links, arranged images in zigzag form|

 

Worked on 16 keywords in the .NET MAUI Maps and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Maps-Money-Keywords.aspx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,65 @@
 # Getting Started with the .NET MAUI Maps Control
 
-A quick-start project that helps you to integrate a MAUI Maps control in a .NET MAUI application. This project contains code to load shapes from a remote file, bind a data source for a shape layer, load a map in a tile layer, and enable basic features like data labels, markers, legends, and tooltips.
- 
-Refer to the following documentation to learn about the .NET MAUI Maps Control: 
-https://help.syncfusion.com/maui/maps/getting-started
-
-Check out this online example of the .NET MAUI Maps Control:
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps
+The [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples) control is a powerful data visualization component that displays statistical information for a geographical area. A quick-start project that helps you to integrate a MAUI Maps control in a .NET MAUI application. This project contains code to load shapes from a remote file, bind a data source for a shape layer, load a map in a tile layer, and enable basic features like data labels, markers, legends, and tooltips.
```
Comment (Sharly-Atieno) [inline README.md]: Use the below UTM Parameter, check in all places
(?utm_source=github&utm_medium=listing&utm_campaign=maui-maps-github-samples)

File: README.md [hunk 2]
```diff
@@ -1,15 +1,65 @@
...
+
+## Zooming and panning
+
+Zoom in any layer for a closer look at a specific region by pinching, scrolling the mouse wheel, or track pad. Pan the map to navigate across regions. Zooming and panning can be enabled for both the shape layer and the tile layer.
+
+## Related Links
+
+[Learn More about .NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
```
Comment (Sharly-Atieno) [inline README.md]: Use the below link for download free trial:
 https://www.syncfusion.com/downloads/maui

File: README.md [hunk 3]
```diff
@@ -1,15 +1,65 @@
...
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/maps/getting-started?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-maps?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
+
+[Suggest a Feature](https://www.syncfusion.com/feedback/maui?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples)
```
Comment (Sharly-Atieno) [inline README.md]: [Suggest a feature or report a bug]

**Review by Sharly-Atieno — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yvone-Atieno) [reply README.md]: updated

File: README.md (hunk 3 above)
Comment (Yvone-Atieno) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md]: updated

**Review by Sharly-Atieno — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**